bundles/org.openhab.binding.haassohnpelletoven/pom.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.1.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.haassohnpelletoven</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: haassohnpelletoven Binding</name>
```

Comment:
Is the capitalization correct?

Reply:
I guess yes. It was generated automatically. I compared it with other bindings. It's similar

Reply:
Can you use the official notation and capitalization?

Reply:
Sorry I dont get it. As I said this was automatically generated by the script as described here:

[https://www.openhab.org/docs/developer/](url)

Reply:
Yeah, but that doesn't mean it's correct, isn't it? What about this:

`<name>openHAB Add-ons :: Bundles :: Haas + Sohn Pallet Oven Binding</name>`